geode-core/src/integrationTest/java/org/apache/geode/cache/query/partitioned/PRIndexStatisticsJUnitTest.java

```diff
@@ -273,8 +273,8 @@ public void testStatsForCompactMapRangeIndex() throws Exception {
       query.execute();
     }
 
-    // Both RangeIndex should be used
-    assertEquals(100 /* Execution time */, keyIndexStats.getTotalUses());
+    // Index should not be used
```

Comment:
Could this comment (and others in this class) be expanded slightly to explain why the index should not be used here? I am assuming it's because the query uses `!=`, but it would be good to make it explicit.